### PR TITLE
Fix prysm tests

### PR DESCRIPTION
**Pull request by 0x00101010**
This is a draft one that would expect big changes or be separated into smaller chunks later. Currently submitted here for QA, conversation purposes mainly.

* Fixing prysm tests in git workflows
* Updating test files due to sharding configuration change (time based vs block based)

### CODE REVIEW

File: prysm/Dockerfile.prysm [hunk 1]
```diff
@@ -19,7 +19,7 @@ RUN go build -o /build/validator -buildvcs=false ./cmd/validator
 RUN go build -o /build/bootnode -buildvcs=false ./tools/bootnode
 
 FROM ubuntu:20.04
-RUN apt-get update && apt-get install -y curl jq iproute2
+RUN apt-get update && apt-get install -y curl jq iproute2 vim
```
Comment (roberto-bayardo) [inline prysm/Dockerfile.prysm:22]: vim?

File: lighthouse/vars.env [hunk 2]
```diff
@@ -33,7 +33,7 @@ GENESIS_DELAY=0
 BOOTNODE_PORT=4242
 
 # Network ID and Chain ID of local eth1 test network
-CHAIN_ID=4242
+CHAIN_ID=1
```
Comment (roberto-bayardo) [inline lighthouse/vars.env:36]: Comment above says this should be chain-id for the "local eth1 test network", so why change it to mainnet?

File: prysm/Dockerfile.prysm (hunk 1 above)
Comment (0x00101010) [reply prysm/Dockerfile.prysm:22]: This is a test for now, need vim to check read / navigate log (execution.log) easier.

Will remove when everything is tested

File: lighthouse/vars.env (hunk 2 above)
Comment (0x00101010) [reply lighthouse/vars.env:36]: prysm / geth currently is set to 1, and not changing it causes test errors.

This related to my other question, does ChainID matter here? Or it is fine as long as we set all those clients to the same ChainID, say 4242 here?

File: prysm/run_beacon_node.sh [hunk 3]
```diff
@@ -30,8 +30,9 @@ beacon-node \
     --datadir /chaindata \
     --force-clear-db \
     --interop-eth1data-votes \
-    --http-web3provider="$EXECUTION_NODE_URL" \
-    --deposit-contract 0x8A04d14125D0FDCDc742F4A05C051De07232EDa4 \
+    --execution-endpoint="$EXECUTION_NODE_URL" \
```
Comment (Inphi) [inline prysm/run_beacon_node.sh:33]: Do you still see issues when using `--http-web3provider`. the auth issues could be related to https://github.com/prysmaticlabs/prysm/pull/11748, but the old flag still works for me.

File: lighthouse/vars.env (hunk 2 above)
Comment (Inphi) [reply lighthouse/vars.env:36]: It doesn't matter what the chain ID is. Though we should use a value that doesn't conflict with existing networks.

**Review by Inphi — COMMENTED** [review]
The latest eip4844 prysm is at https://github.com/prysmaticlabs/prysm/tree/eip4844? The prysm submodule config nees to be updated.

There's an old, incomplete PR that also fixes prysm tests - https://github.com/Inphi/eip4844-interop/pull/78/files. Please take a look at that PR as it handles a couple other issues in the test setup.

**Pull request closed without merging**